en/docs/deploy-and-publish/deploy-on-gateway/api-gateway/message-tracing.md

```diff
@@ -1,5 +1,8 @@
 # Message Tracing
 
+!!! Info "Deprecated feature"
```

Comment:
My understanding is that this page probably gets deleted in 4.0.0. Please check.

Reply:
No, this will not get deleted in 4.0.0. There is a high probability that we will be deleting this from the next release onwards.